libpod/version.go

```diff
@@ -24,10 +26,12 @@ type Version struct {
 	GitCommit string
 	Built     int64
 	OsArch    string
+	RunC      string
+	Conmon    string
 }
 
 // GetVersion returns a VersionOutput struct for varlink and podman
-func GetVersion() (Version, error) {
+func GetVersion(rt *Runtime) (Version, error) {
```

Comment:
func (rt *Runtime) GetVersion() to be consistent with the other Runtime functions

Reply:
no longer applicable